app/code/Magento/Sitemap/Test/Unit/Model/Config/Backend/PriorityTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Sitemap\Test\Unit\Model\Config\Backend;
+
+use Magento\Sitemap\Model\Config\Backend\Priority;
+use PHPUnit\Framework\MockObject\MockObject;
+
+/**
+ * Tests for @see Priority
+ */
+class PriorityTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var Priority|MockObject
+     */
+    private $priorityMock;
+
+    /**
+     * @inheritDoc
+     */
+    protected function setUp()
+    {
+        $this->priorityMock = $this->getMockBuilder(Priority::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getValue'])
+            ->getMock();
+    }
+
+    /**
+     * @param string $value
+     * @dataProvider dataProviderTestBeforeSaveValueCorrect
+     */
+    public function testBeforeSave($value)
```

Comment:
"testBeforeSaveIsChainable" to explain what actually assertion do

Reply:
thanks, done